__tests__/bin/utils/setMiddlewareUtils.spec.js

```diff
@@ -174,7 +176,9 @@ describe('Set Middleware Utils', () => {
       }
       expect(mockFs.linkSync).toHaveBeenCalled();
       expect(mockFs.symlinkSync).not.toHaveBeenCalled();
-      process.platform = originalPlatform;
+      Object.defineProperty(process, 'platform', {
+        value: originalPlatform,
+      });
```

Comment:
why was this necessary?

Reply:
So the way this was written before ```process.platform = originaPlatform``` would make tests fail because I believe that you can't directly change that value Im pretty sure it's read only but you can do it in this way to change the value of process.platform. the test itself changes it to win32 and then back to the original platform.